### PR TITLE
Added duty report - list of reports created while on duty

### LINKED ISSUE #19: When going off duty, get a Duty report
When going off-duty, show reports filed during shift (reuse search results)

### PR DESCRIPTION
**Pull request by lebedynskyi**
When-going-off-duty-get-a-Duty-report
Closes #19

### CODE REVIEW

File: app/src/main/java/org/wildaid/ofish/data/RepositoryImpl.kt [hunk 1]
```diff
@@ -35,6 +37,12 @@ class RepositoryImpl(
     override fun saveOnDutyChange(onDuty: Boolean) {
         realmDataSource.saveOnDutyChange(onDuty)
         androidDataSource.saveOnDutyStatus(onDuty)
+        if (onDuty) {
+            androidDataSource.setOnDutyStartTime(Calendar.getInstance().time.time)
```
Comment (VitaliyGre) [inline app/src/main/java/org/wildaid/ofish/data/RepositoryImpl.kt:41]: You can use just Date().time

File: app/src/main/java/org/wildaid/ofish/data/RepositoryImpl.kt [hunk 2]
```diff
@@ -35,6 +37,12 @@ class RepositoryImpl(
...
+            androidDataSource.setOnDutyEndTime(0)
+        } else {
+            androidDataSource.setOnDutyEndTime(Calendar.getInstance().time.time)
```
Comment (VitaliyGre) [inline app/src/main/java/org/wildaid/ofish/data/RepositoryImpl.kt:44]: You can use just Date().time

File: app/src/main/java/org/wildaid/ofish/ui/home/HomeActivity.kt [hunk 3]
```diff
@@ -78,7 +87,7 @@ class HomeActivity : AppCompatActivity(R.layout.activity_home) {
             getString(android.R.string.cancel)
         ).bundle()
 
-        navigation.navigate(R.id.action_homeFragment_to_ask_change_duty_dialog, dialogBundle)
+        navigation.navigate(R.id.ask_change_duty_dialog, dialogBundle)
```
Comment (VitaliyGre) [inline app/src/main/java/org/wildaid/ofish/ui/home/HomeActivity.kt:90]: Is this renaming needed?

File: app/src/main/java/org/wildaid/ofish/data/RepositoryImpl.kt (hunk 2 above)
Comment (lebedynskyi) [reply app/src/main/java/org/wildaid/ofish/data/RepositoryImpl.kt:44]: Done

File: app/src/main/java/org/wildaid/ofish/data/RepositoryImpl.kt (hunk 1 above)
Comment (lebedynskyi) [reply app/src/main/java/org/wildaid/ofish/data/RepositoryImpl.kt:41]: Done

File: app/src/main/java/org/wildaid/ofish/ui/home/HomeActivity.kt (hunk 3 above)
Comment (lebedynskyi) [reply app/src/main/java/org/wildaid/ofish/ui/home/HomeActivity.kt:90]: Yes. After updating a libraries it started crashing if we use different source point -  Fragment tries to use Activity

**Pull request merged**